src/measure_theory/measure_space.lean

```diff
@@ -831,6 +831,10 @@ by rw [← measure_univ_eq_zero, restrict_apply_univ]
 
 @[simp] lemma restrict_univ : μ.restrict univ = μ := ext $ λ s hs, by simp [hs]
 
+lemma eq_restrict_of_measurable_subset (ht : is_measurable t) (t_subset : t ⊆ s) :
```

Comment:
In general, the left hand side in a lemma should be more complicated than the right hand side, so that one would most often use the direct direction with `simp`  or `rw`. So, can you exchange the two sides (and rename the lemma accordingly)? Same thing for other lemma below, `eq_restrict_of_subset_of_measurable`.